mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import {ReactNode} from "react";
+import clsx from "clsx";
+import {PositionTooltip} from "src/component/tooltip/positionTooltip/PositionTooltip";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+interface TooltipProps {
+  /**
+  * Tooltip's children (ReactNode)
+  */
+  children: ReactNode;
+  /**
+  * Tooltip's content (string | ReactNode)
```

Comment:
why do you duplicate types in comments?

Reply:
what should you write then? Can you post a link that describes rules like documenting code?

Reply:
Just a short description

Reply:
ask @Ekaterina1994 in telegram to sent you a McConnel

Reply:
Done)